packages/react-components/react-alert/src/components/Alert/useAlertStyles.ts

```diff
@@ -1,34 +1,81 @@
-import { makeStyles, mergeClasses } from '@griffel/react';
+import { tokens } from '@fluentui/react-theme';
+import { shorthands, makeStyles, mergeClasses } from '@griffel/react';
+
 import type { AlertSlots, AlertState } from './Alert.types';
 import type { SlotClassNames } from '@fluentui/react-utilities';
-
 export const alertClassName = 'fui-Alert';
 export const alertClassNames: SlotClassNames<AlertSlots> = {
   root: 'fui-Alert',
-  // TODO: add class names for all slots on AlertSlots.
-  // Should be of the form `<slotName>: 'fui-Alert__<slotName>`
+  icon: 'fui-Alert__icon',
+  content: 'fui-Alert__content',
+  action: 'fui-Alert__action',
 };
 
 /**
  * Styles for the root slot
  */
 const useStyles = makeStyles({
   root: {
-    // TODO Add default styles for the root element
+    display: 'flex',
+    alignItems: 'center',
+    minHeight: '44px',
+    backgroundColor: tokens.colorNeutralBackground1, // todo - there is no bg10, us
+    ...shorthands.borderColor('transparent'),
+    ...shorthands.borderRadius('4px'),
+    boxShadow: tokens.shadow8,
+  },
+  icon: {
+    height: '16px',
+    ...shorthands.padding('0', '8px', '0', '12px'),
+  },
+  iconSuccess: {
+    color: tokens.colorPaletteGreenForeground2,
+  },
+  iconError: {
+    color: tokens.colorPaletteRedForeground3,
+  },
+  iconWarning: {
+    color: tokens.colorPaletteYellowForeground1, // todo - there is no foreground
+  },
+  iconInfo: {
+    color: tokens.colorNeutralForeground3,
+  },
+  content: {
+    fontSize: '14px', // todo - what is the size & lineheight in tokens
+    fontWeight: tokens.fontWeightSemibold,
+  },
+  action: {
+    ...shorthands.padding('12px'),
+    minWidth: 0,
+    marginLeft: 'auto',
+    color: tokens.colorBrandForeground2, // todo - foreground3 doesn't exist
   },
-
-  // TODO add additional classes for different states and/or slots
 });
 
 /**
  * Apply styling to the Alert slots based on the state
  */
 export const useAlertStyles_unstable = (state: AlertState): AlertState => {
   const styles = useStyles();
-  state.root.className = mergeClasses(alertClassName, styles.root, state.root.className);
+  state.root.className = mergeClasses(alertClassNames.root, styles.root, state.root.className);
+
+  if (state.icon) {
+    state.icon.className = mergeClasses(
+      alertClassNames.icon,
+      styles.icon,
+      state.intent === 'success' && styles.iconSuccess,
+      state.intent === 'error' && styles.iconError,
+      state.intent === 'warning' && styles.iconWarning,
+      state.intent === 'info' && styles.iconInfo,
```

Comment:
_nit:_ If you create an extra `useIntentStyles` call and just have its keys be `success`, `error`, `warning` and `info` then you can simplify this to be `intentStyles[state.intent]`.

```ts
const useIntentStyles = makeStyles({
  error: { ... },
  info: { ... },
  success: { ... },
  warning: { ... },
});

...

const intentStyles = useIntentStyles();

if (state.icon) {
  state.icon.className = mergeClasses(
    alertClassNames.icon,
    styles.icon,
    intentStyles[state.intent],
    state.icon.className,
  );
}
```